app/interactions/create_jwt_token.rb

```diff
@@ -0,0 +1,27 @@
+class CreateJwtToken < ActiveInteraction::Base
+	SECRET_KEY = Rails.configuration.jwt_secret_key
+
+	string :email
+
+	string :password
+
+	validate :check_user
+
+	def execute
+		return unless @user
+
+		encode
+	end
+
+	private
+
+	def check_user
+		@user = User.find_by( email: email )&.authenticate( password )
+		errors.add :login, 'wrong email or password' unless @user
```

Comment:
unless @user.nil?

Reply:
no, check for true/false does not require check for null